src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,9 +3,11 @@
 import java.util.Random;
 
 public class Lottery {
+    public static final Random RANDOM = new Random();
```

Comment:
This should not be a static, just a regular field  `public  final Random Random = new Random();`

Reply:
the test requires setting the private modifier

Reply:
sure, `private` is needed for the regular class field.